gui/src/app/tinystan/index.ts

```diff
@@ -311,3 +354,12 @@ export default class StanModel {
     return version;
   }
 }
+
+
+const jsonify = (obj: string | object): string => {
```

Comment:
I'd consider calling it `string_safe_jsonify` to clarify why you aren't just using `JSON.stringify()` directly (but it's probably also fine in the file-local context)